src/Html.php

```diff
@@ -568,7 +568,7 @@ public static function redirectToLogin($params = '')
 
         $dest     = $CFG_GLPI["root_doc"] . "/index.php";
 
-        if (!isset($AJAX_INCLUDE)) {
+        if ($AJAX_INCLUDE) {
```

Comment:
This means the inverse than original, no?

Reply:
Indeed, fixed.